packages/tools/pie-icons-react/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@justeattakeaway/pie-icons-react",
+  "version": "0.0.1-alpha.0",
+  "tag": "beta",
+  "main": "esm/index.js",
+  "files": [
+    "dist",
+    "esm"
```

Comment:
Would be good to include the individual icon files here too as part of the bundle, just for any consuming applications that can't tree-shake (just gives more options for how they can consume the package potentially).